cmd/node/main.go

```diff
@@ -723,27 +723,33 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 		log.Error("could not create bootsrapper", "err", err)
 		return err
 	}
+
+	shardCoordinator := genesisShardCoordinator
+	currentEpoch := uint32(0)
+	storerEpoch := currentEpoch
+
 	bootstrapParameters, err := bootstrapper.Bootstrap()
 	if err != nil {
 		log.Error("boostrap return error", "error", err)
```

Comment:
If the node fails to bootstrap (might not gather the p2p connections in the bounded time) it will start from genesis and then stop (as it won't be able to sync from the network - wrong epoch) and the validator's owner won't be able to tell if the validator performed well or not. The return error is a must, at least for now, until we have a retry mechanism.

Reply:
reverted